### PR TITLE
Feature/initial project

### PR DESCRIPTION
**Pull request by matt-wills**
This PR introduces:

* Initial project and FR domain objects
* Mongo documents and repositories
* Converters for converting OB model objects to/from FR domain objects
* Common services and common utilities

This needs to be turned into a SpringBoot and docker project.
After that, the main task is to add all of the controllers for the various versions of the OB Read/Write API.

### CODE REVIEW

**Review by BohoCode — CHANGES_REQUESTED** [review]
All PRs shoudl be tied to an Issue in the SecureBankingAccelratorToolkit project please! This way we can use code to do project managment/release notes etc etc.

**Comment by matt-wills** [comment]
> All PRs shoudl be tied to an Issue in the SecureBankingAccelratorToolkit project please! This way we can use code to do project managment/release notes etc etc.

Sure, I was intending to do that 👍🏻 😄

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.forgerock.openbanking.aspsp</groupId>
```
Comment (jorgesanchezperez) [inline pom.xml:28]: Could it make sense to change the `groupId` to `com.forgerock.sba.aspsp`?

Comment (jorgesanchezperez) [reply pom.xml:28]: The same for the `artifactId`?

**Comment by matt-wills** [comment]
Renaming repo. Declining this PR.

**Pull request closed without merging**